### ISSUE TITLE
Fix the Psychopomp eidolon

### ISSUE THREAD

**Issue by rectulo** [TikaelSol/PF2e-Animal-Companions#28]
```
<p><strong>Soul Guardian</strong> Str 18, Dex 14, Con 16, Int 10, Wis 12, Cha 8; +2 AC (+3 Dex cap) Scribe of</p>
<p><strong>the Dead</strong> Str 12, Dex 18, Con 12, Int 14, Wis 12, Cha 10; +1 AC (+4 Dex cap)</p>
```

should be 
```
<p><strong>Soul Guardian</strong> Str 18, Dex 14, Con 16, Int 10, Wis 12, Cha 8; +2 AC (+3 Dex cap) </p>
<p><strong>Scribe of the Dead</strong> Str 12, Dex 18, Con 12, Int 14, Wis 12, Cha 10; +1 AC (+4 Dex cap)</p>
```

**Comment by rectulo** [comment]
Also 

`@Compendium[pf2e.conditionitems.Hidden]{Hidden} watcher` 

Should be just **Hidden watcher**

**Comment by TikaelSol** [comment]
Closed in v5.10, release expected April 27th.
